tests/integration/docidcache/test_crud_cache.py

```diff
@@ -0,0 +1,215 @@
+import itertools
+import os
+from pathlib import Path
+
+import numpy as np
+import pytest
+
+from jina import Flow, Document
+from jina.executors.indexers import BaseIndexer
+from jina.executors.indexers.cache import DocIDCache
+from jina.executors.indexers.keyvalue import BinaryPbIndexer
+from jina.executors.indexers.vector import NumpyIndexer
+
+cur_dir = Path(os.path.dirname(os.path.abspath(__file__)))
+
+KV_IDX_FILENAME = 'kv_idx.bin'
```

Comment:
related to other message above, seems weird to depend so heavily on having these file names

Reply:
Well, we need to get the right file

Reply:
No if u use `BaseExecutor.load_config` from the `indexer` uses config, it will be able to load from the `bin` automatically

Reply:
I know it is not ideal to trust other features, but in this case I do believe that it simplifies the code drastically